### PR TITLE
Change the token encoded in a file name for envelope type to priority

### PR DESCRIPTION
**Pull request by bidetofevil**
## Goal

Change the token encoded in the payload filename to priority, move it to the front of the file, so the natural sort order of the file name will be the priority order

## Testing
Changed unit tests to verify the new scheme

### CODE REVIEW

**Comment by bidetofevil** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/embrace-io/embrace-android-sdk/1511?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#1511** <a href="https://app.graphite.dev/github/pr/embrace-io/embrace-android-sdk/1511?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#1509** <a href="https://app.graphite.dev/github/pr/embrace-io/embrace-android-sdk/1509?utm_source=stack-comment-icon" target="_blank">Graphite</a>: 1 other dependent PR ([#1510](https://github.com/embrace-io/embrace-android-sdk/pull/1510) <a href="https://app.graphite.dev/github/pr/embrace-io/embrace-android-sdk/1510?utm_source=stack-comment-icon" target="_blank">Graphite</a>)
* **#1498** <a href="https://app.graphite.dev/github/pr/embrace-io/embrace-android-sdk/1498?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#1497** <a href="https://app.graphite.dev/github/pr/embrace-io/embrace-android-sdk/1497?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @bidetofevil and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: embrace-android-core/src/main/kotlin/io/embrace/android/embracesdk/internal/session/orchestrator/V2PayloadStore.kt [hunk 1]
```diff
@@ -48,7 +48,7 @@ internal class V2PayloadStore(
             System.NativeCrash.value -> SupportedEnvelopeType.CRASH
             System.ReactNativeCrash.value -> SupportedEnvelopeType.CRASH
             System.FlutterException.value -> SupportedEnvelopeType.CRASH
-            System.NetworkCapturedRequest.value -> SupportedEnvelopeType.NETWORK
+            System.NetworkCapturedRequest.value -> SupportedEnvelopeType.BLOB
```
Comment (fractalwrench) [inline embrace-android-core/src/main/kotlin/io/embrace/android/embracesdk/internal/session/orchestrator/V2PayloadStore.kt:51]: IMO this terminology is somewhat overloaded with AEI

**Review by fractalwrench — APPROVED** [review]
LGTM

File: embrace-android-core/src/main/kotlin/io/embrace/android/embracesdk/internal/session/orchestrator/V2PayloadStore.kt (hunk 1 above)
Comment (bidetofevil) [reply embrace-android-core/src/main/kotlin/io/embrace/android/embracesdk/internal/session/orchestrator/V2PayloadStore.kt:51]: Yeah fair. These all contain blobs but AEI has a blob within a blob, and soon, crashes will too.

I'll ask the group to see what folks like and we can change it after

**Comment by bidetofevil** [comment]
### Merge activity

* **Oct 11, 4:54 PM EDT**: A user started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/embrace-io/embrace-android-sdk/1511).

**Pull request merged**